.dockerignore

```diff
@@ -0,0 +1,9 @@
+cmake-build-*
+.idea
+daemon
+virtual_vehicle_error_log.*
+VirtualVehicle
+_tmp
```

Comment:
Maybe ignore all dirs which starts with _? (_*)?

Reply:
Added ignoring of all dirs starting with _